### PR TITLE
Fix tests for 2.0b9

### PR DESCRIPTION
**Pull request by ahuang11**
## Summary

Support >=2.0b9

## Checklist
- [ ] Summarized PR's changes in [CHANGELOG.md](https://github.com/PrefectHQ/prefect-gcp/blob/main/CHANGELOG.md)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,6 +19,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Security
 
+## 0.1.2
+Released on July 21st, 2022.
+
+### Changed
+- Updated tests to be compatible with core Prefect library (v2.0b9) and bumped required version - [#30](https://github.com/PrefectHQ/prefect-gcp/pull/30)
+
```
Comment (desertaxle) [inline CHANGELOG.md:27]: I don't think we need a new release for this. I think #31 will be a reason to release.

Comment (ahuang11) [reply CHANGELOG.md:27]: Yeah, I was planning to get 31 inside here and release together

**Review by desertaxle — DISMISSED** [review]

**Comment by desertaxle** [comment]
Let's do a separate PR to get the changelog all squared away before we release.

**Comment by ahuang11** [comment]
I think #31 has the changelog all ready. Do you mean you want to remove `## 0.1.2` from these PRs and in a separate PR add it back?

**Comment by desertaxle** [comment]
I think there might be conflicts so I'm going to ignore change log entries for these PRs. We'll see what it looks like once these are merged and clean up if necessary. In the future, I think we should have specific PRs that just update the changelog for release to prevent conflicts and confusion.

**Comment by ahuang11** [comment]
Should I merge the Blocks one first then?

**Comment by desertaxle** [comment]
Yeah, that will probably avoid potential conflicts.

**Review by desertaxle — APPROVED** [review]

**Pull request merged**